sql/connect/server/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectServiceSuite.scala

```diff
@@ -873,8 +873,11 @@ class SparkConnectServiceSuite
     val listenerBus = sparkContext.listenerBus
     val LISTENER_BUS_TIMEOUT = 30000
     def executeHolder: ExecuteHolder = {
-      assert(listener.executeHolder.isDefined)
-      listener.executeHolder.get
+      // An ExecuteHolder will eventually be set
+      Eventually.eventually(timeout(1.seconds)) {
```

Comment:
If we need this, `1.seconds` is enough? Could you provide the failure error message or track trace which you want to fix in this PR?

Reply:
That makes sense. I'll change this and line 894 to 10 to seconds + add an error message.